src/taco-cli/test/create.ts

```diff
@@ -227,14 +227,15 @@ describe("taco create", function (): void {
         templateManager = new TemplateManager(kitHelper);
 
         // Delete existing run folder if necessary
-        rimraf(runFolder, function (err: Error): void {
-            if (err) {
-                done(err);
-            } else {
-                // Create the run folder for our tests
-                wrench.mkdirSyncRecursive(runFolder, 511); // 511 decimal is 0777 octal
-                done();
-            }
+        rimraf.sync(runFolder);
+
+        // Create the run folder for our tests
+        wrench.mkdirSyncRecursive(runFolder, 511); // 511 decimal is 0777 octal
+
+        // Load taco-kits error codes
+        tacoPackageLoader.lazyRequire("taco-kits", "taco-kits").then((tacoKits: typeof TacoKits) => {
+            tacoKitsErrors = tacoKits.TacoErrorCode;
+            done();
```

Comment:
I believe that you can not take `done` as an argument, and just return this promise here. You could even promise chain the rimraf and wrench together with the lazy require (or do them all concurrently)